snippets/ztp_example-sno.yaml

```diff
@@ -0,0 +1,81 @@
+# example-node1-bmh-secret & assisted-deployment-pull-secret need to be created under same namespace example-sno
+---
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "example-sno"
+  namespace: "example-sno"
+spec:
+  baseDomain: "example.com"
+  cpuPartitioningMode: AllNodes
```

Comment:
Considering this is Tech Preview, do we really want to provide it in this example?
As the very least, we should point out that a support exception will be required

Reply:
We do point out that this is a Tech Preview feature in a number of places. More obvious in the HTML output.